components/home/Banner.tsx

```diff
@@ -70,26 +69,30 @@ function MobileBanner() {
     <Flex
       justify={'end'}
       direction={'column'}
-      w={'24.125rem'}
-      h={'23.25rem'}
-      mt={'24px'}
-      px={'2rem'}
-      py={'1.5rem'}
+      w={'95%'}
+      h={'96'}
```

Comment:
Generally, never give exact height to a div unless absolutely needed. You can achieve the same result by using padding & margins